proposal/press_release.md

```diff
@@ -0,0 +1,13 @@
+Jupyter releases a new Telemetry System
+
+Today the Jupyter Team announced the release of its newest feature—the Jupyter Telemetry System. This new system enables anyone deploying Jupyter products to safely collect structured “event” data from Jupyter applications. This makes it easier to secure Jupyter deployments, detect intrusion, improve user experience, scale resources effectively, and minimize costs for both administrators and users. It achieves all of this while prioritizing user privacy. The result is an improved Jupyter experience for all. 
+
+Jupyter’s telemetry system has been a long time coming. Members of the Jupyter community have been exploring various ways to collect structured events inside Jupyter applications. It’s no easy task. While event data can improve the development and deployment of Jupyter applications, it’s difficult to create an effective, safe and secure event-logging system. 
+
+This new telemetry system strikes a successful balance between effective event-logging and honoring user privacy. Transparency is a key tenant. Users are always informed when data is being collected. In most cases, users can “opt-in” and dictate which events are logged. In stricter Jupyter environments where users cannot dictate the event-logging (e.g. FEDRAMP, FERPA, HIPAA, GDPR-compliant services), users will be informed by clear and transparent messages from Jupyter’s telemetry system. 
+
+While staying user-friendly, the telemetry system is both flexible and extensible for administrators. It integrates seamlessly with Jupyter-based applications and extensions and can be extended to work with non-Jupyter applications. It is highly configurable—allowing administrators to dictate what events are collected and when. It can be configured to store code executed on Jupyter Kernels, track clicks by users inside JupyterLab, record which datasets have been opened. For highly secure, multi-user environments, this new system can be configured to track usernames or personal identifiable information (PII); it has already been tested in highly regulated settings (see NYU’s ADRF and XX). 
+
+The system provides necessary building blocks for telemetry. There are four major components: 1) an extensible JSON event schema, 2) a Python API for integrating Python-based applications with the telemetry system, 3) a Javascript API for integrating JS-based applications with the telemetry system and 4) a HTTP REST API for integrating non-Jupyter applications with telemetry system. Together, these components can be rearranged and configured to work with any deployment.
```

Comment:
I would make this a bit more generic since the technical details are bound to change (they're already different from what is in this PR vs the JEP) 

Something like 
```
The telemetry system provides an extensible event schema, APIs and libraries for Jupyter server, JupyterLab, JupyterHub and Jupyter Classic, and a flexible configuration system to work with any deployment.
```

Reply:
👍 that sounds good. I'll revamp this paragraph for generality.